src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -221,11 +222,30 @@ def _create_database_account(client,
         public_network_access = 'Enabled' if enable_public_network else 'Disabled'
 
     system_assigned_identity = None
+    UserAssignedIdentitiesValue = cmd.get_models('UserAssignedIdentitiesValue')
+    SYSTEM_ID = '[system]'
+    enable_system = False
     if assign_identity is not None:
         if assign_identity == [] or (len(assign_identity) == 1 and assign_identity[0] == '[system]'):
+            enable_system = True
             system_assigned_identity = ManagedServiceIdentity(type=ResourceIdentityType.system_assigned.value)
         else:
-            raise InvalidArgumentValueError("Only '[system]' is supported right now for command '--assign-identity'.")
+            set_user_identities = {}
+            for x in assign_identity:
+                if x != SYSTEM_ID:
+                    set_user_identities[x] = UserAssignedIdentitiesValue()
+                else:
+                    enable_system = True
+            if enable_system:
+                system_assigned_identity = ManagedServiceIdentity(
+                    type=ResourceIdentityType.system_assigned_user_assigned.value,
```

Comment:
format the type=xxx into type = xxx

Reply:
According to the linter, these spaces should be removed in function parameters.